test/unit/test_webdrivermonkeypatches.py

```diff
@@ -1,69 +0,0 @@
-import unittest
-
-from mockito import when
-
-from selenium.webdriver.remote.webelement import WebElement
-from selenium.webdriver.remote.webdriver import WebDriver as RemoteWebDriver
-
-
-SCRIPT = "return [ window.id, window.name, document.title, document.URL ];"
-HANDLE = "17c3dc18-0443-478b-aec6-ed7e2a5da7e1"
-
-
-class MockWebDriver(RemoteWebDriver):
-    def __init__(self):
-        pass
-
-    current_window_handle = HANDLE
-
-
-class WebDriverMonkeyPatchesTests(unittest.TestCase):
-
-    def test_window_info_values_are_strings(self):
-        driver = MockWebDriver()
-        when(driver).execute_script(SCRIPT).thenReturn(['id', 'name', 'title', 'url'])
-        info = driver.get_current_window_info()
```

Comment:
I am be missing something here but it looks like all these unit tests just test get_current_window_info method which I don't see being removed above. I can see removing these unit tests if get_current_window_info is removed but as it is not yet is this premature to remove these unit tests?